Paws4Me/Paws4Me/Controllers/Logic/SignInViewController.swift

```diff
@@ -11,20 +11,27 @@ protocol Validation {
 }
 
 class SignInViewController: UIViewController, Validation {
+    // MARK: - IBOulets
     @IBOutlet weak private var txtUsername: UITextField!
     @IBOutlet weak private var txtPassword: UITextField!
     @IBOutlet weak private var btnSignIn: UIButton!
     @IBOutlet weak private var btnRegister: UIButton!
+
+    // MARK: - Vars/Lets
     private var isLoggedIn = false
     private let bottomLine = CALayer()
     private let bottomLine2 = CALayer()
+
+    // MARK: - Life cycle
     override func viewDidLoad() {
         super.viewDidLoad()
         btnSignIn.addCornerRadius(btnSignIn)
         btnSignIn.addCornerRadius(btnRegister)
         btnSignIn.changeBorderLook(btnRegister)
         addUnderline(txtUsername, txtPassword, bottomLine: bottomLine, bottomLine2: bottomLine2)
     }
+
+    // MARK: - IBActions
     @IBAction private func signInButtonTapped (_ sender: UIButton!) {
         guard let username = txtUsername.text else { return }
```

Comment:
Just some code clean up, you could have a combined guard let. Some thing like this 
 guard username = txt, name = txt else {
    return
  }

Reply:
Thank you Simekani for this. I never knew you could combine guard statements. 👍🏻